spark-batch-python/app-package/src/main/resources/oozie/example/workflow.xml

```diff
@@ -20,7 +20,7 @@
             <jar>${environment_name_node}${component_hdfs_root}/lib/job.py</jar>
             <spark-opts>--jars hdfs:///user/deployment/platform/spark-examples.jar --py-files /opt/pnda/platformlibs-py2.7.egg</spark-opts>
             <arg>${component_input_data}</arg>
-            <arg>${hdfspath_output_path}</arg>
+            <arg>/user/${wf:user()}/${hdfspath_output_path}</arg>
```

Comment:
This looks like it would expand to: /user/username//parquet/${component_application}. Note double //. What do you think?

Reply:
Good catch. I pushed the hfs.json change too, which removes the root '/' and defines a relative path instead now.